sql/src/main/java/io/crate/operation/projectors/SortingProjector.java

```diff
@@ -126,4 +106,35 @@ public void fail(Throwable t) {
         }
         return requirements;
     }
+
+    @Nullable
+    @Override
+    public Function<BatchIterator, Tuple<BatchIterator, RowReceiver>> batchIteratorProjection() {
+        return bi -> {
+            Collector<Row, ?, Bucket> collector = Collectors.mapping(
+                this::getObjects,
+                Collectors.collectingAndThen(Collectors.toList(), this::sortAndCreateBucket));
+            return new Tuple<>(CollectingBatchIterator.newInstance(bi, collector), downstream);
+        };
+    }
+
+    private Object[] getObjects(Row row) {
```

Comment:
hm we used the term `cells` for that before afaik, wouldn't it be maybe more clear? anyway `objects` is a bit too generic to understand the method immediately imho